src/Workspaces/Core/Portable/EmbeddedLanguages/Json/JsonKind.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.Json
+{
+    internal enum JsonKind
+    {
+        None,
```

Comment:
Is this used? If not, maybe `None = 0` or remove?

Reply:
i like this as it makes it easy to test if a token is default.

Reply:
Should it be `None = 0` then?

Reply:
Sure.   i can do that.

Reply:
#resolved.